pkgs/by-name/da/dashy-ui/package.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  yarnConfigHook,
+  yarnBuildHook,
+  nodejs,
+  yq-go,
+  settings ? { },
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dashy-ui";
+  # This is like 3.1.1 but the latest working yarn.lock.
+  # All other changes are for docs with the exception of 768d746cbfcf365c58ad1194c5ccc74c14f3ed3a, which simply adds no-referrer meta tag
+  version = "3.1.1-unstable-2024-07-1";
```

```suggestion
  version = "3.1.1-unstable-2024-07-14";
```
Doh, looks like this got cut off.

---

Ah yes - I'll fix that when I'm back on my laptop.